onnx/defs/reduction/defs.cc

```diff
@@ -221,7 +221,8 @@ If keepdims equal 0, then the resulting tensor have the reduced dimension pruned
 If select_last_index is True (default False), the index of the last occurrence of the {name} 
 is selected if the {name} appears more than once in the input. Otherwise the index of the 
 first occurrence is selected.
-The type of the output tensor is integer.)DOC";
+The type of the output tensor is integer.
+complement_axis needs be used simultaneously with axis attribute. If complement_axis is True (default False), it will reduce other axes except the target axis (complementarily).)DOC";
```

Comment:
Suggested rewording: "The attribute complement_axis is used in conjunction with the axis attribute, and has a default value of False. If complement_axis is True, all axes except the specified axis will be reduced."